Source/Libraries/GSF.Security/packages.config

```diff
@@ -2,4 +2,6 @@
 <packages>
   <package id="Antlr" version="3.5.0.2" targetFramework="net46" />
   <package id="ExpressionEvaluator" version="2.0.4.0" targetFramework="net46" />
+  <package id="JWT" version="8.2.3" targetFramework="net46" />
```

Comment:
Now that we have added two new dependencies `GSF.Security` all dependent downstream application installers will need to add new DLLs. `Netwonsoft.Json` should already be in most systems, any that depend on `GSF.Web` anyway, but I think `JWT` is new.

Please verify that the `Netwonsoft.Json` version matches what is deployed with `GSF.Web`.

Also, will need to add `JWT` to the added dependencies in the monthly security report.

Reply:
Removed JWT Dependency to simplify Dependency Handling.

Reply:
Updated Newtonsoft to match `GSF.Web`